go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/SUSE/connect-ng
 
-go 1.16
+go 1.21
```

Comment:
This is actually a big change. Has this been handled in packaging?

Reply:
yes this is part of: https://bugzilla.suse.com/show_bug.cgi?id=1219185
I has been approved and we also changed it already in the spec file. This is more or less a remnant.